src/platform/plugins/shared/visualizations/public/embeddable/visualize_embeddable.tsx

```diff
@@ -313,7 +308,13 @@ export const getVisualizeEmbeddableFactory: (deps: {
           },
         ],
         savedObjectProperties: getUnchangingComparator(),
-        linkedToLibrary: [linkedToLibrary$, (value) => linkedToLibrary$.next(value)],
+        linkedToLibrary: [
+          linkedToLibrary$,
+          (value) => linkedToLibrary$.next(value),
+          (a, b) => {
+            return a === undefined || b === undefined ? true : a === b;
+          },
+        ],
```

Comment:
This looks to be different from https://github.com/elastic/kibana/pull/210125 - was this intentional?

Reply:
Its a combination of https://github.com/elastic/kibana/pull/210125 and https://github.com/elastic/kibana/pull/211264.

This is similar to what is merged into [8.16](https://github.com/elastic/kibana/pull/211057).